src/replays/replayitem.py

```diff
@@ -122,7 +122,18 @@ def update(self, message, client):
         
         self.name      = message["name"]
         self.mapname   = message["map"]
-        self.duration  = time.strftime('%H:%M:%S', time.gmtime(message["duration"]))
+        if message['end'] == 4294967295:  # = FFFF FFFF (year 2106) aka still playing
+            seconds = time.time()-message['start']
+            if seconds > 86400:  # more than 24 hours
+                self.duration = str(int(seconds/86400)) + " days<br />&nbsp;! error !"
+            elif seconds > 7200:  # more than 2 hours
+                self.duration = time.strftime('%H:%M:%S', time.gmtime(seconds)) + "<br />? error ?"
```

Comment:
if you don't want to alarm people, why are you putting "! error !" there? I agree that it's not a critical problem for the client that should lead to the client shutting down, but putting "! error !" or "? error ?" into the display just sucks.